spec/exercises.js

```diff
@@ -44,18 +50,23 @@ describe('Testing exercises', function () {
   var exercises = exec.sync(['list', '--lang=en']).toString().split('\n')
   exercises.pop() // last line break
   exercises.map(function (exercise) {
-    return wUtil.idFromName(exercise)
-  }).forEach(function (id, nr) {
-    var folder = path.join(process.cwd(), 'test', id)
-    var allFiles = require('glob').sync('*.*', {
+    return {
+      name: exercise,
+      id: wUtil.idFromName(exercise, config.spaceChar)
+    }
+  }).forEach(function ({ name, id }, nr) {
+    var folder = path.join(process.cwd(), config.exercisesFolder || 'test', id)
+    var allFiles = require('glob').sync(config.files || '*.*', {
       cwd: folder
     })
 
     allFiles.filter(function (file) {
-      return /^(in)?valid(-|_)?(\d*)?\.\w+/.test(file)
+      var validRegex = config.validRegex || /^(in)?valid(-|_)?(\d*)?\.\w+/
```

Comment:
This config is related to https://github.com/workshopper/workshopper-adventure-test/blob/8399a69bf9ca57c53323d0d349806ef1f44230e8/spec/exercises.js#L74 this line alone is a problematic config. If you want to create a config for this, it might need to be `{ validRegex?: RegExp, invalidRegex?: RegExp }` where both are tested.